benchmark/parse.php

```diff
@@ -1,6 +1,9 @@
 <?php
 
-require __DIR__.'/../vendor/autoload.php';
+require __DIR__.'/../src/Exception/ParserException.php';
+require __DIR__.'/../src/Exception/MissingIdnSupport.php';
+require __DIR__.'/../src/Parser/UriParser.php';
+require __DIR__.'/../src/functions.php';
```

Comment:
Why?

Reply:
Because when using webgrind the composer stuffs get in the way of the result.